components/announcement_bar/overage_users_banner/index.tsx

```diff
@@ -43,7 +43,7 @@ const OverageUsersBanner = () => {
     const license = useSelector(getLicense);
     const seatsPurchased = parseInt(license.Users, 10);
     const isCloud = useSelector(isCurrentLicenseCloud);
-    const getPreferencesCategory = makeGetCategory();
+    const getPreferencesCategory = useMemo(() => makeGetCategory(), []);
```

Comment:
This works. I think we could also simplify to `useMemo(makeGetCategory, []);`

Reply:
Oh yeah, I guess that would. I'll change since it's simpler